src/main/java/org/jitsi/videobridge/Videobridge.java

```diff
@@ -1037,29 +1035,33 @@ else if (sctpConn == null)
                                 XMPPError.Condition.bad_request,
                                 "No SCTP connection found for ID: " + id);
                     }
+                }
 
-                    // expire
-                    if (expire
-                            != ColibriConferenceIQ.Channel.EXPIRE_NOT_SPECIFIED)
+                // expire
+                if (expire
+                    != ColibriConferenceIQ.Channel.EXPIRE_NOT_SPECIFIED)
+                {
+                    if (expire < 0)
                     {
-                        if (expire < 0)
-                        {
-                            return IQUtils.createError(
-                                    conferenceIQ,
-                                    XMPPError.Condition.bad_request,
-                                    "Invalid 'expire' value: " + expire);
-                        }
+                        return IQUtils.createError(
+                            conferenceIQ,
+                            XMPPError.Condition.bad_request,
+                            "Invalid 'expire' value: " + expire);
+                    }
 
-                        // Check if SCTP connection has expired.
-                        if (expire == 0 && sctpConn.isExpired())
-                            continue;
+                    sctpConn.setExpire(expire);
```

Comment:
is this the real fix? moving this line above the if check?

Reply:
This one attempts to fix the problem of an expired (expire=0) sctpconnection being included in the response. I think it's appropriate since we hande channels the same way:
https://github.com/jitsi/jitsi-videobridge/blob/master/src/main/java/org/jitsi/videobridge/Videobridge.java#L866

Reply:
Seems to work fine here. The SctpConnection is removed now.

I am seeing this in the logs: `WARNING: SCTP connection with NA-lTs2sSgimlkjgGybgXQ|||||APMUI_-LJEOiqKtXwQ19WoQ not ready yet.` Might be related.